commit c37fce5b3019c7a150203fc3a484885591b194de
Author: Alexis Christoforides <alexis@thenull.net>
Date:   Sun Dec 2 00:10:24 2018 -0500

    Help Path.IsPathRooted method overload selection.

    .NET Core, and Mono after merging https://github.com/mono/mono/pull/11342, introduce ambiguity with a new overload.

Index: xamarin-pkg-fsharp/src/scripts/scriptlib.fsx
===================================================================
--- xamarin-pkg-fsharp.orig/src/scripts/scriptlib.fsx
+++ xamarin-pkg-fsharp/src/scripts/scriptlib.fsx
@@ -93,7 +93,7 @@ module Scripting =
 
     module Process =
 
-        let processExePath baseDir exe =
+        let processExePath baseDir (exe:string) =
             if Path.IsPathRooted(exe) then exe
             else 
                 match Path.GetDirectoryName(exe) with
